css/style-landing.css

```diff
@@ -7,42 +7,50 @@
 }
 
 .home-menu {
-    padding: 0.5em;
+    padding: 0.4em;
     text-align: center;
 }
 .home-menu {
     background: black;
 }
 
-
 .home-menu .pure-menu-item {
     font-size: 12px;
-    letter-spacing: 5px;
+    letter-spacing: 2px;
 }
 
 .home-menu a {
     color: white;
 }
-.home-menu li a:hover,
-.home-menu li a:focus {
+.home-menu li a:hover {
     background: black;
     color: gray;
 }
 
 .top-img {
     position: absolute;
-    margin-left: auto;
-    margin-right: auto;
-    top: 30%;
-    left: 50%;
-    width: 500px;
+    margin: auto;
+    left:0;
+    right: 0;
+    top: 0;
+    bottom: 0;
+    width: 600px;
     height: auto;
-    margin-left: -250px;
     z-index: 5;
 }
 
-.bot-img {
-    width: 100%;
-    height: 100%;
-    position: relative;
-}
\ No newline at end of file
+#bg-vid {
+    position: fixed;
+    top: 50%;
+    left: 50%;
+    min-width: 100%;
+    min-height: 100%;
+    width: auto;
+    height: auto;
+    z-index: -100;
+    -webkit-transform: translateX(-50%) translateY(-50%);
+    transform: translateX(-50%) translateY(-50%);
+    -webkit-transition: 1s opacity;
```

Comment:
Also, it doesn't look like this actually fades or has altered opacity... btw I recommend using an overlay with opacity to hide the low quality of the vid